inspirehep/modules/literaturesuggest/static/js/literaturesuggest/mapper.js

```diff
@@ -73,18 +73,24 @@ define(function(require, exports, module) {
      * @param depositionType {String} type of deposition
      * @returns {*}
      */
-    map: function(data, depositionType) {
+    map: function(data) {
+      if (data.type === 'journal-article'){
```

Comment:
So instead of having here hardcoded values from `crossref` you did not find a way to do it in `doi.js` ?